src/cluster.c

```diff
@@ -672,13 +672,13 @@ void clusterInit(void) {
         serverLog(LL_WARNING, "No bind address is configured, but it is required for the Cluster bus.");
         exit(1);
     }
-    int cport = server.cluster_port ? server.cluster_port : port + CLUSTER_PORT_INCR;
-    if (listenToPort(cport, &server.cfd) == C_ERR ) {
+    if (!server.cluster_port) server.cluster_port = port + CLUSTER_PORT_INCR;
+    if (listenToPort(server.cluster_port, &server.cfd) == C_ERR ) {
```

Comment:
i happen to skim through this PR yesterday and noticed there was a new INFO field, which i suppose is the reason for this change, but i notice the new info field was now dropped, so i wonder if this change is still needed.

same about the change in getInfoProperty (which is good, but could be conflicting with another PR)

Reply:
the change is no longer necessary, i can remove it (i thought it would be a good cleanup, like we dose not maintain the server.cluster_port value). ok, i decided to remove it, unrelated, and reduce diff

the change in getInfoProperty is needed, see https://github.com/redis/redis/pull/10344#discussion_r915418042
or should i use the old one? (but i guess it could still conflicting with the other one)

Reply:
if you need it keep keep the new one.. the conflict will be resolved when the time comes.